lib/Command/Train.php

```diff
@@ -130,7 +137,7 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 			$config = $config->setLearningRate((float)$input->getOption('learn-rate'));
 		}
 
-		$trainingDataConfig = TrainingDataConfig::default();
+		$trainingDataConfig = TrainingDataConfig::default((int) $input->getOption('now'));
```

Comment:
Not so important but what if the provided option value is not convertible to an int?

Reply:
Then :boom: I guess

For this development-only switch I don't care too much tbh